app/assets/javascripts/_page_show.es6

```diff
@@ -58,11 +58,40 @@
       });
     }
 
+    // Remediate internal links on PageBuilder paragraph and accordion components
+    function remediateInternalLinksTarget(){
+        let intLinks = $('.page-paragraph-component, .page-accordion-component').find("a[href*='marketplace.va.gov'], a[href^='/'],a[href^='.']");
```

Comment:
Question: Do we have any existing urls that start with `.`? Just curious about the use case for that.

Reply:
Yeah, it's a weird thing Brad found during testing. If you create an internal URL (`/about`) via TinyMCE editor, it'll create a `../../about` in the markup.